### DISCUSSION TITLE
Is it possible to indicate a base folders to search for .code-workspace files?

### DISCUSSION THREAD

**Discussion by carloscadux** [alefragnani/vscode-project-manager#595] [Q&A] [answered]
Hi,
I'd appreciate your help.
I explored the extension settings, and found several options to add base folders according to different projects, including folder projects.
However, I couldn't notice whether it is possible to indicate a base folder to search for **.code-workspace files**.
Is it possible to set a folder that would list all **.code-workspace files** under it?
Thanks!

**Comment by alefragnani** [comment, marked as answer]
Hi @carloscadux ,

This is not supported yet, but there is an open issue requesting this https://github.com/alefragnani/vscode-project-manager/issues/334.

Feel free to upvote and/or add comments, for easier tracking.

Hope this helps

**Comment by carloscadux** [comment]
Thanks! I already left my comment on thread https://github.com/alefragnani/vscode-project-manager/issues/334.
